docs/content/guides/developer/app-examples/tic-tac-toe.mdx

```diff
@@ -265,11 +265,11 @@ public fun create_game(x_addr: address, o_addr: address, ctx: &mut TxContext) {
         id: object::new(ctx),
```

Comment:
This is the closest I can get to L252, where there's a `uid_to_inner` that could be turned into a receiver function call.